src/MessagingBubble/DeletedMessagingBubble.less

```diff
@@ -8,16 +8,9 @@
   flex-direction: row;
 }
 
-.DeletedMessagingBubble--Message--ownMessage {
+.DeletedMessagingBubble--Message {
   .border--m(@neutral_silver);
-  max-width: 16.125rem;
-  border-radius: 0.3125rem; // 5px
-  color: @neutral_medium_gray;
-}
-
-.DeletedMessagingBubble--Message--otherMessage {
-  .border--m(@neutral_silver);
-  max-width: 18.4375rem;
```

Comment:
hmm where'd the own vs other styling go?

Reply:
The differentiated bit is now in `DeletedMessagingBubble--Container--<own/other>Message`, and the identical bit is now in `DeletedMessagingBubble--Message`.